.cirrus.yml

```diff
@@ -94,11 +93,14 @@ task:
       sysctl --system
       ;;
     centos-stream-8)
-      yum install -y -q dnf-plugins-core
-      yum config-manager --set-enabled powertools
+      yum config-manager --set-enabled powertools # for glibc-static
       ;;
     esac
-    yum install -y -q gcc git iptables jq glibc-static libseccomp-devel make criu
+    # Work around dnf mirror failures by retrying a few times.
+    for i in $(seq 0 2); do
+      sleep $i
+      yum install -y -q gcc git iptables jq glibc-static libseccomp-devel make criu && break
```

Comment:
We should make sure to fail here when yum failed on i==2

Reply:
Assuming the script is run with `set -e`, it will fail if yum fails on the last iteration. In case `-e` is not set, well, we have to add it to all scripts.

Reply:
Yes, it's run with `set -e`. Yes we need to check exit code after `for` 🤦🏻 

Thanks for catching this! Fixed, PTAL.